src/rust/src/x509/sct.rs

```diff
@@ -139,13 +254,19 @@ pub(crate) fn parse_scts(
         let log_id = sct_data.read_exact(32)?.try_into().unwrap();
         let timestamp = u64::from_be_bytes(sct_data.read_exact(8)?.try_into().unwrap());
         let _extensions = sct_data.read_length_prefixed()?;
-        let _sig_alg = sct_data.read_exact(2)?;
-        let _signature = sct_data.read_length_prefixed()?;
+        let hash_algorithm =
+            u8::from_be_bytes(sct_data.read_exact(1)?.try_into().unwrap()).try_into()?;
+        let signature_algorithm =
+            u8::from_be_bytes(sct_data.read_exact(1)?.try_into().unwrap()).try_into()?;
```

Comment:
These can just be `sct_data.read_byte()?.try_into()?` right?

Reply:
I think so, yes. Will change.